conbench/app/_plots.py

```diff
@@ -917,7 +917,7 @@ def get_source_for_single_benchmark_result(current_benchmark_result, cur_run, un
     # clearly on the benchmark result object we can remove the `cur_run`
     # argument and have type checking find potential bugs.
 
-    if not cur_run["commit"]["timestamp"]:
+    if not cur_run["commit"] or not cur_run["commit"]["timestamp"]:
         return None, None
```

Comment:
Right! No commit for no means "no timestamp", i.e. no way to position this result on the plot.

(consideration does not apply in the other way of plotting -- which we use in c-bench tinyplots for example -- where we pick the position in time using the benchmark start time)

Reply:
Also relates to #1489 but now I see that's a small subset of #633!